Source/WebCore/platform/graphics/texmap/coordinated/CoordinatedPlatformLayer.cpp

```diff
@@ -383,8 +401,8 @@ void CoordinatedPlatformLayer::setContentsOpaque(bool contentsOpaque)
         return;
 
     m_contentsOpaque = contentsOpaque;
-    m_nicosia.delta.flagsChanged = true;
-    // FIXME: request a full repaint.
+    m_pendingChanges.add(Change::ContentsOpaque);
+    // FIXME: request a full repaint?
```

Comment:
This is scary, maybe add a bug directly. Probably we need a full repaint request, even if it is a performance penalty.

Reply:
We used to do that, but I'm not sure it's needed, maybe setNeedsDisplay is already called when ContentsOpaque changes. So, I removed it on purpose to see if anything fails without it.